src/resources/Cookie/CookieMethods.ts

```diff
@@ -13,3 +15,22 @@ export function getCookie(cname: string) {
   }
   return '';
 }
+
+export const useCookieListener = (cookieName: string, interval = 2000) => {
+  const [cookie, setCookie] = useState(getCookie(cookieName));
+  const [displayAlert, setDisplayAlert] = useState(false);
+
+  useEffect(() => {
+    const checkCookie = setInterval(() => {
+      const currentCookie = getCookie(cookieName);
+      if (currentCookie !== cookie) {
+        setCookie(currentCookie);
+        setDisplayAlert(true);
+      }
+    }, interval); // Check every interval
+
+    return () => clearInterval(checkCookie);
+  }, [cookie, cookieName]);
+
+  return displayAlert;
```

Comment:
I'm not sure if this will be used for anything else than displaying an alert, but in the context of a "useCookieListener"-function, should "displayAlert" also have a more generic name like "cookieChanged" or "cookieUpdateDetected"?

Reply:
I agree - I'll change it 🫡